src/component/admin/billing/BillingGrid/BillingGrid.tsx

```diff
@@ -0,0 +1,325 @@
+import { FC, useMemo } from 'react';
+import { SxProps } from '@mui/system';
+import { Alert, Divider, Grid, styled, Theme, Typography } from '@mui/material';
+import { Link } from 'react-router-dom';
+import CheckIcon from '@mui/icons-material/Check';
+import { colors } from 'themes/colors';
+import useUsers from 'hooks/api/getters/useUsers/useUsers';
+import useInvoices from 'hooks/api/getters/useInvoices/useInvoices';
+import { BillingInfoButton } from './BillingInfoButton/BillingInfoButton';
+import { BillingHistory } from './BillingHistory/BillingHistory';
+import { useInstanceStatus } from 'hooks/api/getters/useInstanceStatus/useInstanceStatus';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { InstanceState } from 'interfaces/instance';
+
+const GridRow: FC<{ sx?: SxProps<Theme> }> = ({ sx, children }) => {
+    return (
+        <Grid
+            container
+            item
+            justifyContent="space-between"
+            alignItems="center"
+            sx={{ flexWrap: 'nowrap', ...sx }}
+        >
+            {children}
+        </Grid>
+    );
+};
+
+const GridCol: FC<{ vertical?: boolean }> = ({
+    children,
+    vertical = false,
+}) => {
+    return (
+        <Grid
+            container={vertical}
+            item
+            display="flex"
+            alignItems={vertical ? 'start' : 'center'}
+            direction={vertical ? 'column' : undefined}
+        >
+            {children}
+        </Grid>
+    );
+};
+
+const GridColLink: FC = ({ children }) => {
+    return (
+        <Typography
+            sx={theme => ({
+                fontSize: theme.fontSizes.smallBody,
+                marginLeft: '8px',
+            })}
+            component="span"
+        >
+            ({children})
+        </Typography>
+    );
+};
+
+export const BillingGrid: FC = () => {
+    const { instanceStatus } = useInstanceStatus();
+    const { users } = useUsers();
+    const { invoices } = useInvoices();
+
+    const price = {
```

Comment:
This is fine for now, but I think eventually we might want to get this from an API. Might come up in the next iteration.